fetcher.go

```diff
@@ -169,8 +169,24 @@ func (f *fetcherSession) loader(ctx context.Context) ipld.Loader {
 	}
 }
 
-func prototypeFromLink(lnk ipld.Link) (ipld.NodePrototype, error) {
-	return dagpb.AddDagPBSupportToChooser(func(_ ipld.Link, lnkCtx ipld.LinkContext) (ipld.NodePrototype, error) {
-		return basicnode.Prototype__Any{}, nil
-	})(lnk, ipld.LinkContext{})
+func DefaultPrototypeChooser(lnk ipld.Link, lnkCtx ipld.LinkContext) (ipld.NodePrototype, error) {
+	c, ok := lnk.(cidlink.Link)
+	if ok {
+		switch c.Cid.Prefix().Codec {
+		case 0x70:
+			return dagpb.Type.PBNode, nil
```

Comment:
you can use https://github.com/ipld/go-codec-dagpb/commit/b641b0ff139c9342c4de8a84bafc015422708662 now.

you also don't need the raw special case, as basicnode.Prototype.Any will already work fine for raw nodes, picking Bytes.

Reply:
I made this change.